pkg/google/gcp.go

```diff
@@ -131,7 +136,9 @@ func New(config *Config) (*GCP, error) {
 				DefaultDiscount: config.DefaultDiscount,
 			}, cloudCatalogClient, regionsClient, storageClient)
 			if err != nil {
-				log.Printf("Error creating GCS collector: %s", err)
+				logger.LogAttrs(ctx, slog.LevelWarn, "Error creating collector",
```

Comment:
Maybe should be:

```suggestion
				logger.LogAttrs(ctx, slog.LevelError, "Error creating collector",
```

?

Reply:
Ah yes, we should. That's definitely an error and not a warning